openfisca_france/scripts/parameters/baremes_ipp/README.md

```diff
@@ -0,0 +1,66 @@
+# Barèmes IPP
+
+## Présentation
+
+IPP = [Institut des politiques publiques](http://www.ipp.eu/en/)
+
+L'IPP produit des fichiers au format XLSX appelés « Barèmes IPP » :
+- en anglais : http://www.ipp.eu/en/tools/ipp-tax-and-benefit-tables/
+- en français : http://www.ipp.eu/fr/outils/baremes-ipp/
+
+Ces barèmes IPP sont très complets et il est intéressant pour OpenFisca de profiter de leur contenu.
+
+## Conversion XLSX vers XML
+
+OpenFisca définit ses paramètres dans un format XML décrit [ici](https://doc.openfisca.fr/legislation-parameters/index.html).
```

Comment:
This URL will change once https://github.com/openfisca/openfisca-doc/pull/72 is merged.

Should be `https://doc.openfisca.fr/coding-the-legislation/legislation_parameters.html` I think.